python/ray/tests/mpi/test_mpi.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import ray
+
+
+def test_mpi_func_pi(ray_start_regular):
+    @ray.remote(
+        runtime_env={
+            "mpi": {
+                "args": ["-n", "4"],
+                "worker_entry": "mpi_worker.py",
```

Comment:
Where does it find the file? From the current directory?

Reply:
I think we should be clear about it in the docstring

Reply:
I think it's from the working dir. I'll add the doc.